Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1418,6 +1665,239 @@ def run_script_command():
+    timeout = args.get('timeout')
+    timeout_duration = args.get('timeout_duration')
+
+    timeout = timeout and int(timeout)
+    response = run_batch_get_cmd(host_ids, file_path, optional_hosts, timeout, timeout_duration)
+
+    resources: dict = response.get('combined', {}).get('resources', {})
+
+    output = []
+
+    for _, resource in resources.items():
+        errors = resource.get('errors', [])
+        if errors:
+            error_message = errors[0].get('message', '')
+            if not error_message:
+                error_message = f'Could not get command\n{errors}'
+            return_error(error_message)
+        output.append({
+            'HostID': resource.get('aid'),
+            'Stdout': resource.get('stdout'),
+            'Stderr': resource.get('stderr'),
+            'BaseCommand': resource.get('base_command'),
+            'TaskID': resource.get('task_id'),
+            'GetRequestID': response.get('batch_get_cmd_req_id'),
+            'Complete': resource.get('complete') or False,
+            'FilePath': file_path
+        })
+
+    human_readable = tableToMarkdown(f'Get command has requested for a file {file_path}', output)
+    entry_context = {
+        'CrowdStrike.Command(val.TaskID === obj.TaskID)': output
+    }
+
+    return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def status_get_command():
+    args = demisto.args()
+    request_ids = argToList(args.get('request_ids'))
+    timeout = args.get('timeout')
+    timeout_duration = args.get('timeout_duration')
+
+    timeout = timeout and int(timeout)
+
+    responses = []
+    files_output = []
+    file_standard_context = []
+
+    for request_id in request_ids:
+        response = status_get_cmd(request_id, timeout, timeout_duration)
+        responses.append(response)
+
+        resources: dict = response.get('resources', {})
+
+        for _, resource in resources.items():
+            errors = resource.get('errors', [])
+            if errors:
+                error_message = errors[0].get('message', '')
+                if not error_message:
+                    error_message = f'Could not get command\n{errors}'
+                return_error(error_message)
+            files_output.append({
+                'ID': resource.get('id'),
+                'TaskID': resource.get('cloud_request_id'),
+                'CreatedAt': resource.get('created_at'),
+                'DeletedAt': resource.get('deleted_at'),
+                'UpdatedAt': resource.get('updated_at'),
+                'Name': resource.get('name'),
+                'Size': resource.get('size'),
+                'SHA256': resource.get('sha256')
+            })
+            file_standard_context.append({
+                'Name': resource.get('name'),
+                'SHA256': resource.get('sha256'),
+                'Size': resource.get('size'),
+            })
+
+    human_readable = tableToMarkdown('CrowdStrike Falcon files', files_output)
```

Comment:
please update title here

Reply:
What do you mean by updating title. Do you mean changing 'CrowdStrike Falcon files' to something else? Please let me know why the title is not good.